### ISSUE TITLE
[Accessibility] lacking req. minimum color contrast ratio for options in <VSCodeDropdown> component

### ISSUE THREAD

**Issue by Krituhar** [microsoft/vscode#227533]
- VS Code Version: 1.88.0
- OS Version: MacOS Sonoma (v14.6.1)

**Issue**: We have used VSCode's webview view to create a new pane for our feature. We have used the **<VSCodedropdown>** component from `web-ui-toolkit` which does not meet the required minimum color contrast ratio with its foreground and background text. The contents in the dropdown has a very low color contrast ratio of less than 3.00:1, which is not sufficient as per accessibility standards.

 - **Required contrast ratio for this content: 4.50:1**

**[Recommendation]**
Ensure active user interface components have sufficient contrast.

**Comment by meganrogge** [comment]
A screenshot would help
